posthog/api/session_recording.py

```diff
@@ -136,19 +138,51 @@ def delete(self, request: request.Request, *args: Any, **kwargs: Any) -> Respons
 
         return Response({"success": True})
 
+    @action(methods=["GET"], detail=True)
+    def snapshot_file(self, request: request.Request, **kwargs) -> HttpResponse:
+        file_key = request.GET.get("blob_key")
+
+        if not file_key:
+            raise exceptions.ValidationError("Must provide a snapshot file blob key")
+
+        url = object_storage.get_presigned_url(file_key)
```

Comment:
What's stopping me from getting a different teams files? This should be checked to only contain characters we allow (I guess numbers for now) and the rest of the file key we should build ourselves based on the team and session_id